build.rs

```diff
@@ -1,3 +1,3 @@
 fn main() {
     gear_wasm_builder::build();
-}
+}
```

Comment:
Please configure your editor to add an empty EOL at the end of the file.

VS Code: `"files.insertFinalNewline": true`

```suggestion
}

```